frontend/src/components/cluster/Overview.tsx

```diff
@@ -77,6 +77,7 @@ function EventsSection() {
   const [isWarningEventSwitchChecked, setIsWarningEventSwitchChecked] = React.useState(
     Boolean(localStorage.getItem(EVENT_WARNING_SWITCH_FILTER_STORAGE_KEY))
   );
+  const [events, eventsError] = Event.useList();
```

Comment:
we use `ResourceTable` down below that fetches the events again, can we change that to something like `MainInfoSection` so it uses this same events